src/geocat/comp/climatology.py

```diff
@@ -322,3 +324,142 @@ def month_to_season(
     # Therefore, we must filter out the NaNs
     return means.sel(
         {time_coord_name: means[time_coord_name].dt.month == months[1]})
+
+
+def calendar_average(
+        dset: typing.Union[xr.DataArray, xr.Dataset],
+        freq: str,
+        time_dim: str = None,
+        climatology: bool = False) -> typing.Union[xr.DataArray, xr.Dataset]:
+    """This function computes averages according to a given time frequency.
+
+    Parameters
+    ----------
+    dset : `xarray.Dataset`, `xarray.DataArray`
```

Comment:
I don't think the backticks are needed when using numpydoc style 

```suggestion
    dset : xarray.Dataset, xarray.DataArray
```

Reply:
Backticks aren't required, but if you specify the argument type in the docstring like this ```:class: `xarray.DataArray` ``` it is turned into a hyperlink that leads to the documentation page for that class, object, etc. I just forgot to add `:class:` before it and `:obj:` before `str` and `boolean`. Currently the documentation pages for geocat-comp are inconsistent in whether or not they use this feature. I'm hoping to go through and rectify that at some point.